selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/FindText.java

```diff
@@ -506,4 +511,21 @@ public void openFileNodeByDoubleClick(String pathToFile) {
         .click();
     actionsFactory.createAction(seleniumWebDriver).doubleClick().perform();
   }
+
+  /** get number of found occurrences in a page */
```

Comment:
I suggest you to create data value class to encapsulate each number:
```
SearchFileResult(String result)
-----------------------
private final int foundOnPageNumber;
private final int occurrenceNumber;
private final int totalFoundNumber;
-----------------------
public int getFoundOnPageNumber()
public int getOccurranceNumber()
public int getTotalFoundNumber()
-----------------------

Reply:
This object **SearchFileResult** being returning from the dedicated method of **FindText** class allows to get all required data by one method call, and to have logic to parse the search results string in one most suitable place.